bitnami/vault/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: vault
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/vault
-version: 1.0.4
+version: 1.0.5
```

New feature is added, according to [semver](https://semver.org), a new minor version should be bumped.
```suggestion
version: 1.1.0
```

---

Sure I will update chart version